Automattic/studio
# Studio: Add animation when we receive a message UI

Fixes https://github.com/Automattic/dotcom-forge/issues/7679

## Proposed Changes

This PR adds animations when the user receives a new message.

Also, the following changes have implemented:
1. Remove the last message if there is an error
2. Add a "conversation" feeling by delaying the assistant thinking
3. Move thinking to be a message instead of passing it as a separate component. That ensures that there is no "bump" effect when thinking is added/removed.

## Testing Instructions



1. Open Studio and navigate to AI Assistant tab
4. Send a message
5. When you see the message, make sure there is an animation when it appears, like so:
6. 

https://github.com/Automattic/studio/assets/497103/74d5e810-13eb-4607-bd83-187f49e59e13



## Pre-merge Checklist
